editor-components/call_to_action/call_to_action7/call_to_action7.tsx

```diff
@@ -86,54 +43,41 @@ class CallToAction7Page extends BaseCallToAction {
   }
 
   render() {
-    const styling = {
-      backgroundColor: this.getPropValue("cardBackgroundColor"),
-    };
+    const title = this.getPropValue("title") as string;
```

Comment:
that's not a string value. please remove "as string". this method will return a JSX element